cell_locking.hh

```diff
@@ -320,15 +320,19 @@ private:
     static constexpr size_t compute_rehash_at_size(size_t bucket_count) {
         return bucket_count * max_load_factor::num / max_load_factor::den;
     }
-    void maybe_rehash() {
+    void maybe_rehash() noexcept {
         if (_partition_count >= _rehash_at_size) {
+          try {
             auto new_bucket_count = std::min(_partitions.bucket_count() * 2, _partitions.bucket_count() + 64 * 1024);
             auto buckets = std::make_unique<partitions_type::bucket_type[]>(new_bucket_count);
 
             _partitions.rehash(partitions_type::bucket_traits(buckets.get(), new_bucket_count));
             _buckets = std::move(buckets);
 
             _rehash_at_size = compute_rehash_at_size(new_bucket_count);
+          } catch (...) {
+            // Ignore any error
```

Comment:
It's obvious you're ignoring errors, the role of the comment is to explain why that's okay.

Reply:
clarified comment and moved to top of function in v2